policyengine_us/variables/gov/hhs/medicaid/costs/medicaid_per_capita_cost.py

```diff
@@ -0,0 +1,81 @@
+from policyengine_us.model_api import *
+from policyengine_us.variables.gov.hhs.medicaid.costs.medicaid_group import (
+    MedicaidGroup,
+)
+
+
+class medicaid_per_capita_cost(Variable):
+    value_type = float
+    entity = Person
+    label = "Per capita Medicaid cost by eligibility group & state"
+    unit = USD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        state = person.household("state_code", period)
+        group = person("medicaid_group", period)
+
+        p = parameters(period).calibration.gov.hhs.medicaid
+
+        # Handle each group type separately
+        is_aged_disabled = group == MedicaidGroup.AGED_DISABLED
+        is_child = group == MedicaidGroup.CHILD
+        is_expansion_adult = group == MedicaidGroup.EXPANSION_ADULT
+        is_non_expansion_adult = group == MedicaidGroup.NON_EXPANSION_ADULT
+
+        # Calculate spend and enrollment for each group
+        aged_spend = p.spending.by_eligibility_group.aged[state]
+        disabled_spend = p.spending.by_eligibility_group.disabled[state]
+        child_spend = p.spending.by_eligibility_group.child[state]
+        expansion_adult_spend = (
+            p.spending.by_eligibility_group.expansion_adults[state]
+        )
+        non_expansion_adult_spend = (
+            p.spending.by_eligibility_group.non_expansion_adults[state]
+        )
+
+        aged_enroll = p.enrollment.aged[state]
+        disabled_enroll = p.enrollment.disabled[state]
+        child_enroll = p.enrollment.child[state]
+        expansion_adult_enroll = p.enrollment.expansion_adults[state]
+        non_expansion_adult_enroll = p.enrollment.non_expansion_adults[state]
+
+        # Combine aged and disabled for AGED_DISABLED group
+        aged_disabled_spend = aged_spend + disabled_spend
+        aged_disabled_enroll = aged_enroll + disabled_enroll
+
+        # Select the right values based on group
+        spend = select(
+            [
+                is_aged_disabled,
+                is_child,
+                is_expansion_adult,
+                is_non_expansion_adult,
+            ],
+            [
+                aged_disabled_spend,
+                child_spend,
+                expansion_adult_spend,
+                non_expansion_adult_spend,
+            ],
+            default=0.0,
+        )
+
+        enroll = select(
+            [
+                is_aged_disabled,
+                is_child,
+                is_expansion_adult,
+                is_non_expansion_adult,
+            ],
+            [
+                aged_disabled_enroll,
+                child_enroll,
+                expansion_adult_enroll,
+                non_expansion_adult_enroll,
+            ],
+            default=1.0,  # Prevents divide by zero for NONE group
+        )
+
+        # Avoid divide‑by‑zero in non‑expansion states, etc.
+        return where(enroll == 0, 0.0, spend / enroll)
```

Comment:
i think - check it with another file
```suggestion
        mask = enroll > 0
        result = np.zeros(length=mask)
        return (spend / enroll)[mask]
```